lib/internal/errors.js

```diff
@@ -1876,6 +1877,7 @@ E('ERR_WORKER_PATH', (filename) =>
   ) +
   ` Received "${filename}"`,
   TypeError);
+E('ERR_WORKER_SAME_THREAD', 'Cannot connect to the same thread', Error);
```

Comment:
An error message like "Cannot sent a message to the same thread" would likely be better

Reply:
Yes, this was a leftover of the previous PR. Thanks for noticing!